From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: corymhall <43035978+corymhall@users.noreply.github.com>
Date: Mon, 20 Apr 2026 10:20:48 -0400
Subject: [PATCH] Add go.mod to vendored github.com/f5devcentral/go-bigip
 modules


diff --git a/vendor/github.com/f5devcentral/go-bigip/f5teem/go.mod b/vendor/github.com/f5devcentral/go-bigip/f5teem/go.mod
new file mode 100644
index 00000000..a9a76b8b
--- /dev/null
+++ b/vendor/github.com/f5devcentral/go-bigip/f5teem/go.mod
@@ -0,0 +1,8 @@
+module github.com/f5devcentral/go-bigip/f5teem
+
+go 1.13
+
+require (
+	github.com/google/uuid v1.1.1
+	github.com/satori/go.uuid v1.2.0 // indirect
+)
diff --git a/vendor/github.com/f5devcentral/go-bigip/go.mod b/vendor/github.com/f5devcentral/go-bigip/go.mod
new file mode 100644
index 00000000..508d569d
--- /dev/null
+++ b/vendor/github.com/f5devcentral/go-bigip/go.mod
@@ -0,0 +1,10 @@
+module github.com/f5devcentral/go-bigip
+
+go 1.20
+
+require github.com/stretchr/testify v1.2.1
+
+require (
+	github.com/davecgh/go-spew v1.1.0 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+)
